README.md

```diff
@@ -88,3 +88,21 @@ Copyright (c) 1991-2020 by [the GMT Team](AUTHORS.md).
 GMT is released under the
 [GNU Lesser General Public License](http://www.gnu.org/licenses/lgpl.html)
 version 3 or any later version. See [LICENSE.TXT](LICENSE.TXT) for full details.
+
+## Acknowledgment
+
+GMT relies on several other Open Source software libraries and programs for its
+operation.  We gratefully acknowledge the importance to GMT of these products.
+GMT may be linked with these libraries (some are optional):
+[Geospatial Data Abstraction Library (GDAL)](https://gdal.org),
+[Network Common Data Form (netCDF)](https://www.unidata.ucar.edu/software/netcdf/),
+[Perl Compatible Regular Expressions (PCRE)](https://www.pcre.org), 
+[Fastest Fourier Transform in the West (FFTW)](http://www.fftw.org),
```

Comment:
We cannot say that we rely on FFTW (we don't). FFTW is GPL, we don't rely on GPL libs.

Reply:
Agree that we dont rely on it, we may use it.  I dont understand the point about GPL and that we dont rely on GPL libs.  What do you mean?

Reply:
If we say we rely on a GPL lib, GMT has to be GPL.

Reply:
what is the approved verb then? use?

Reply:
Please help phrase this so we can get it done.

Reply:
I think it will do if we add an [optional] at the end of all of them but netCDF.

Reply:
OK, I marked most as optional.  OK with you?

Reply:
> OK, I marked most as optional. OK with you?

It's not me. I don't give a shit on it. It's the GNU zealots that I'm concerned with.

Reply:
It seems the standard practice is to ignore this stuff until a zealots starts attacking.  Then one can add both GPL and LPGL licenses to GMT and users can select which one, apparently.
Seems OK to you @seisman?

Reply:
OK to me.